website/docs/basics/webpack.md

```diff
@@ -0,0 +1,110 @@
+---
+id: webpack
+title: Webpack
+sidebar_label: Webpack
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs)
+
+At its core, Webpack is a static module bundler for modern JavaScript applications. When Webpack processes your application, it internally builds a dependency graph that maps every module your project needs and generates one or more bundles. Learn more about Webpack [here](https://webpack.js.org/concepts/).
+
+To put this simply, Webpack takes your development files and creates a production or development version of them based on the config.
```

Comment:
This sentence feels odd. `Takes development files and creates development version of them`
Can this be modified somehow?

Reply:
It's kind of tricky. I'd just remove the part about creating development version, just to avoid confusion.

```suggestion
To put this simply, Webpack takes your development files and creates a production-ready version of them based on the config.
```